pills/03-enter-environment.xml

```diff
@@ -4,5 +4,385 @@
       version="5.0"
       xml:id="enter-environment">
 
-<title>enter environment</title>
+  <title>enter environment</title>
+
+  <para>
+    Welcome to the third Nix pill. In the previous <link
+    linkend="install-on-your-running-system">second pill</link> we have
+    installed Nix on our running system. Now we can finally play with it a
+    little, things also apply to NixOS users.
+  </para>
+
+  <section>
+    <title>Enter the environment</title>
+
+    <para>
+      In the previous pill we created a nix user, so let's start by switching
+      user with <literal>su - nix</literal>. If your
+      <literal>~/.profile</literal> got evaluated, then your should now be able
+      to run commands like <literal>nix-env</literal> and
+      <literal>nix-store</literal>.
```

Comment:
Several of these `<literal>`s could be `<command>`s or `<filename>`s (throughout the file.)